### DISCUSSION TITLE
Internal links to an index page from other folders are grayed out?

### DISCUSSION THREAD

**Discussion by carlos-truong** [ObsidianPublisher/obsidian-github-publisher#327] [Q&A — Template]
Hi everybody,

When I add internal links to an index page from other folders, they are grayed out after pushing to the github repo. Is there any way to fix it? Or I shouldn't add internal links to index pages?

Here's my Obsidian Folder Structure:
MyPersonalBlogs/FolderA/index.md (it contains the link grayed out: [[../FolderB/index|Title] )
MyPersonalBlogs/FolderB/index.md

**Comment by carlos-truong** [comment]
I don't know why but not only index pages, the internal links to some other pages are not grayed out but the preview window is not displayed correctly. 
For some pages, the preview window is white empty or it shows the bottom of the page instead of the top.

So I want to disable the preview feature. How can I do that?

I'm still using Mkdocs template

**Comment by carlos-truong** [comment]
I've changed the internal links from relative path to absolute path and they are okay now.
But I still want to disable the file preview window because it doesn't work properly for some pages.

**Comment by Mara-Li** [comment]
I won't provide any support for Mkdocs.

**Reply by carlos-truong** [reply]
Okay I got it but do you have an option to disable the file preview?

**Reply by carlos-truong** [reply]
Currently, I just need a way to disable the file preview window while hovering on internal links :D
